docs/development.md

```diff
@@ -21,43 +21,48 @@ For detailed information about `craig-stars` architecture, check out the [archit
 
-## Assets
-You will need art assets, and those can be downloaded from [https://craig-stars.net/images/images.zip](https://craig-stars.net/images/images.zip). Copy these images into the `frontend/static/images` folder.
+After cloning the repo (easily done with Github Desktop), build `craig-stars` locally once. This will build the frontend and then the backend. The backend embeds the frontend resources into its binary to support a single binary deployment. 
 
-## Building and Running
+### Installing go-sqlite3
+If you don't already have it, you'll also need to install [go-sqlite3](https://github.com/mattn/go-sqlite3) for things to work. However, this itself requires the [GCC compiler collection](https://gcc.gnu.org/) to function. 
 
-```bash
-make build
-```
+For Linux/Mac, you can simply install the latest version of the GCC compilers using whatever package software you installed previously.
 
-Install air for automatic server restarts while developing: 
+However, on Windows, you'll need to install a Linux-like development interface (use [MinGW](https://www.mingw-w64.org/) - Cygwin64 WILL NOT WORK) to install & build the latest GCC version.
 
-```bash
-go install github.com/air-verse/air@latest
-```
+Once that's done, enter `go install github.com/mattn/gosqlite3` into your computer's native terminal (or the terminal in VS Code) to install the driver.
+
+## Assets
+You will need art assets, and those can be downloaded from [https://craig-stars.net/images/images.zip](https://craig-stars.net/images/images.zip). Copy these images into the `frontend/static/images` folder (or make a new folder if it doesn't already exist).
+
+## Building and Running
+Finally, go to your terminal and enter the following commands:
 
-Launch the frontend and backend at the same time with `make`
 ```bash
+go install github.com/air-verse/air@latest
+make build
 make dev
+go run main.go serve --debug
 ```
 
+(Installing air is only required once, so you can skip the first line on subsequent builds.)
 **Note** On first launch, this will create an empty database with a single `admin` user, password `admin`.
 
-Point your browser at [http://localhost:5173](http://localhost:5173) to see a live reloading frontend proxied to the go server on port `:8080`. Updating go code will relaunch the backend automatically. Updating frontend code will do a hot reload with sveltekit/vite.
+If done correctly, it should give a localhost link (http://localhost:XXXX/) representing the application being hosted locally on your machine. Go to that site to see a live reloading frontend proxied to the go server on port `:8080`. Updating go code will relaunch the backend automatically, while updating frontend code will do a hot reload with sveltekit/vite. (For obvious reasons, this only lasts while the terminal is actually _open_; closing it will close the program.)
```

Comment:
Does it not run on port 5173 by default on windows?

Reply:
```suggestion
If done correctly, it should give a localhost link (http://localhost:XXXX/) representing the application being hosted locally on your machine. Go to that site to see a live reloading frontend proxied to the go server on port `:8080`. Updating go code will relaunch the backend automatically, while updating frontend code will do a hot reload with sveltekit/vite.
```

Reply:
AFAIK it does run on port 5173 automatically on windows - IDK about linux though.